transform/snowflake-dbt/models/finance/account_daily_arr.sql

```diff
@@ -1,6 +1,7 @@
 {{config({
     "materialized": "table",
-    "schema": "finance"
+    "schema": "finance",
```

Comment:
This model is also used in `marketing_funnel`, which is owned by sales. Shall we ping them?

Reply:
@ifoukarakis , `marketing_channel` sources data from `marketing_gsheets` which ingestion is turned off at the moment, so I think we are ok.